src/types.mligo

```diff
@@ -140,7 +140,12 @@ type plist_direction = bool
 let prev = false
 let next = true
 
-// Proposal Doubly Linked List
+(*
+ * Proposal Doubly Linked List
+ *
+ * Behave like a unique element list (or a set ordered by its insertion time.)
+ * When trying to insert an element that already exists, the list remains unchanged.
```

Comment:
>Behave like a unique element list (or a set ordered by its insertion time.)

I think it is better to just say an [OrderdSet](https://hackage.haskell.org/package/ordered-containers-0.2.2/docs/Data-Set-Ordered.html)?

>  * When trying to insert an element that already exists, the list remains unchanged.

But this is wrong. Right? AFAIU If you try to insert an existing element, then the data structure becomes corrupt.

Reply:
Fixed the description.